wxt_cidrbot/git_webhook_handler.py

```diff
@@ -121,11 +125,15 @@ def webhook_request(self, event):
             if state == 'approved':
                 self.send_approved_message(installation_id, json_string)
 
-    def triage_issue(self, installation_id, json_string, x_event_type):
+    def triage_issue(self, installation_id, json_string, x_event_type, converted_from_draft=False):
         event_info = self.check_installation(installation_id)
         issue_type = "Pull request"
         query_key = "pull_request"
 
+        pull_request_action_msg = "created in"
+        if converted_from_draft:
+            pull_request_action_msg = "has been converted from draft in"
+
```

Comment:
This is probably better written as an if/else - if you need to do some other stuff later on based on the conditions, you won't have to re-do any of the logic flow.